eq/deflect/eventHandler.cpp

```diff
@@ -36,6 +36,56 @@ namespace
 {
 typedef std::vector< EventHandler* > EventHandlers;
 static lunchbox::PerThread< EventHandlers > _eventHandlers;
+
+// Values come from QtCore/qnamespace.h, but don't want to depend on Qt just for that
```

Comment:
We need a generic event library? ;-) No strong opinion, but a list of symbolic names would be quite long and I'm not sure what we gain, as the qnamespace.h is easy to find for a Deflect user (as Deflect depends on Qt). Even if deflect provides the symbolic names you still have to do the following conversion to Equalizer key codes, so that would just serve to make this switch/case a bit more pretty...